src/components/TaskCard/EditTaskCard.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import { editTask, completeTask } from "store/actions";
+import "components/TaskCard/index.scss";
+import {
+  ICON_COMPLETE,
+  COMPLETE_ICON_ALT_TEXT,
+  ICON_DELETE,
+  DELETE_ICON_ALT_TEXT,
+  KEY_ENTER,
+} from "utils/constant";
+
+const EditTaskCard = ({ id, task, onEditableTasks }) => {
```

Comment:
is a single task being edited or multiple tasks?

```suggestion
const EditTaskCard = ({ id, task, onEditTask }) => {
```

Reply:
I'm using this for toggle task editing. renamed it to onToggleTaskEditing. thank you.